examples/inferencing/running-on-gke.md

```diff
@@ -0,0 +1,185 @@
+## Deployment
+
+The deployment to GKE is currently detached from the training pipeline. The model artifact that is a result of training will need to be manually deployed to the cluster using these instructions.
+
+
+### Create an Environment file
```

Comment:
We need a prerequisite section here. e.g. gcloud installation (or direct them to use cloud shell). Also add gcloud commands that are needed to enable services that will be used by this deployment (e.g. GKE, GCS, etc...)